extensions/notion/src/components/PageListItem.tsx

```diff
@@ -118,17 +118,17 @@ export function PageListItem({
   };
 
   const openInNotionAppAction = createOpenInNotionAction("App", "notion-logo.png", () =>
-    handleOnOpenPage(page, setRecentPage, "app"),
+    handleOnOpenPage(page, setRecentPage),
   );
   const openInNotionBrowserAction = createOpenInNotionAction("Browser", Icon.Globe, () =>
-    handleOnOpenPage(page, setRecentPage, "web"),
+    handleOnOpenPage(page, setRecentPage),
   );
 
   const { open_in } = getPreferenceValues<Preferences>();
-  const isDefaultNotionActionApp = open_in === "app";
+  const isDefaultNotionActionApp = open_in && open_in.name === "Notion";
```

Comment:
If people previously selected `app`, maybe it's a good idea to also have it in the if sentence here?

Reply:
Maybe I am missing something here and you can help me but I am not sure how to handle this, since `open_in` is now using the local app `name` and `bundleId` and we have notion set as default, so the same thing as if they had picked`app`.

`open_in` needs `.name` and the value will always be an app name and not the previous `app` or `web` options.

Are you saying that the previous "app" value is still going to persist and I would need to handle it like this:

```
  if (open_in === "app") open_in.name = "Notion";
 ```
 The only issue would be the `open_in` Type `Application` and I'm trying to check for a `String`